lib/include/tick/hawkes/model/base/model_hawkes.h

```diff
@@ -10,7 +10,7 @@
 /** \class ModelHawkes
  * \brief Base class of Hawkes models
  */
-class DLL_PUBLIC ModelHawkes : public Model {
+class DLL_PUBLIC ModelHawkes : public TModel<double> {
```

Comment:
Isn't default `Model` class already `TModel<double>`?

Reply:
This is a forgotten "git checkout"